programs/github

```diff
@@ -1,10 +1,10 @@
 local github = dofile("apis/github")
 local passed = {...}
```

Comment:
Maybe this should be `{select(2, ...)}`, so as to automatically skip `selectAction`, which lets you remove the suspect condition I comment on below